jax/_src/pallas/triton/pallas_call_registration.py

```diff
@@ -58,9 +58,10 @@ def _pallas_call_ttir_lowering(
     num_stages: int,
 ):
   # TODO(sharadmv): Handle multiple devices with different capabilities.
-  d, *_ = jax.local_devices(backend="gpu")
+  # d, *_ = jax.local_devices(backend="gpu")
```

Comment:
This is pending a decision what to do in the short-term and long-term for the compute capabilities